docs/contributors_agreement_explained.rst

```diff
@@ -62,7 +62,7 @@ Do I have to sign the contributor's agreement to submit a patch to the Plone cor
 
   We enthusiastically welcome patches,
   but we can't merge them until you sign and return a contributor's agreement.
-  (Unless, in the judgement of the Plone Release Manager, the patch is so tiny as not to constitute a "creative work".
+  (Unless, in the judgment of the Plone Release Manager, the patch is so tiny as not to constitute a "creative work".
```

Comment:
both spellings are correct

Reply:
So what is the issue here then ?

Reply:
I guess it was a 'If it ain't broke, why fix?' but up to you